libs/styles/typography.css

```diff
@@ -154,14 +161,33 @@
   width: 100%;
 }
 
+.dark .con-button {
+  border-color: #fff;
+  color: #fff;
```

Comment:
Could `--color-white` not be used here as well?

Reply:
yea, might as well. I updated em. Thanks